snomed-ct-integration/src/test/java/dev/ikm/tinkar/snomedct/integration/DescriptionSemanticIT.java

```diff
@@ -38,6 +50,36 @@ public static void setup() {
     public static void shutdown() {
         PrimitiveData.stop();
     }
+    @Test
+    public void testDescriptionSemantics()throws IOException {
+        String sourceFilePath ="C:\\Users\\patrichards\\Solor\\SnomedCT_InternationalRF2_PRODUCTION_20240201T120000Z\\Full\\Terminology\\sct2_Description_Full-en_INT_20240201.txt";
+
+        try(BufferedReader br = new BufferedReader(new FileReader(sourceFilePath))){
+            String line;
+            while((line = br.readLine()) != null){
+                if (line.startsWith("id")) continue;
+                String[] columns = line.split("\\t");
+
+                //pass these args in assertion method
+                long effectiveTimeToLong=dateStringToEpochMillis(columns[1]);
+                EntityProxy.Concept descriptionStatus = Integer.parseInt(columns[2]) == 1 ? TinkarTerm.ACTIVE_STATE : TinkarTerm.INACTIVE_STATE;
+                EntityProxy.Concept descriptionType = SnomedUtility.getDescriptionType(columns[6]);
+                String term =columns[7];
+                EntityProxy.Concept caseSensitivityConcept = SnomedUtility.getDescriptionCaseSignificanceConcept(columns[8]);
```

Comment:
Nice -- I didn't know this method existed. But, we don't want to rely on anything from the snomedct-transformer repo since this repo will replace it. You're better off just copying the method into this class.